examples/analytics-vendors.amp.html

```diff
@@ -430,6 +431,24 @@
 </amp-analytics>
 <!-- End Cxense example -->
 
+<!-- Deep.BI tracking -->
+<amp-analytics type="deepbi" id="deepbi">
+<script type="application/json">
+{
+  "vars": {
+    "datasetKey": "aBcDeFgHiJkL",
+    "accessKey": "aBcDeFgHiJkLMn0123456789",
+    "gdpr":"true"
+  },
+  "extraUrlParams": {
+    "dimension1": "value1"
+  },
```

Comment:
The example didn't work because of invalid json. Please remove the trailing comma here 😄

Reply:
Thank you very much...
Not my best :[